paperqa/settings.py

```diff
@@ -580,6 +584,35 @@ class Settings(BaseSettings):
         frozen=True,
     )
 
+    callbacks: Mapping[str, list[Callable[["EnvironmentState"], Any]]] = Field(
```

Comment:
@nadolskit sorry I missed this before, but could we move this into "AgentSettings" since these only correspond to agent tool calls?

Reply:
done!